openedx/core/djangoapps/credit/services.py

```diff
@@ -21,7 +21,7 @@ def _get_course_key(course_key_or_id):
     where the CourseKey will simply be returned
     """
     return (
-        CourseKey.from_string(course_key_or_id)
+        course_key_from_string_or_404(course_key_or_id)
```

Comment:
is this an appropriate place to use this helper? Is it only being called by a view?